app/views/cases/sar/_close_form.html.slim

```diff
@@ -3,7 +3,7 @@
   .form-group
     = f.gov_uk_date_field :date_responded, { legend_text: t('.close_date'),
             form_hint_text: t('.date_example'),
-            today_button: {class: 'button-secondary'} }
+            today_button: {class: 'acts-like-button'} }
```

Comment:
why are we using this css class?

Reply:
I needs to be a link not a button
Happy to change it to something else

Reply:
You could try just an empty string.